bootstrap

```diff
@@ -3,23 +3,10 @@
 # This script sets up a brand-new FreeBSD host as the proper piece of Buildbot
 # infrastructure.  See README.rst
 #
-# Usage: ./bootstrap HOST
+# Usage: ./bootstrap
 
 set -e
 
-HOSTNAME="${1}"
-if [ -z "${HOSTNAME}" ]; then
-    echo "USAGE: ./bootstrap hostname"
-    exit 1
-fi
+env ASSUME_ALWAYS_YES=YES pkg install ansible git python2
```

Comment:
If this is run in the repo, then git is probably already installed?

Reply:
I tried to structure the script so it could be run using `curl | bash`. It's not completely there because it requires the rest of the repository, so you'd need to at least download a tarball.